rpc/eth_api.go

```diff
@@ -452,13 +455,14 @@ func (e *PublicEthAPI) doCall(args CallArgs, blockNr rpc.BlockNumber, globalGasC
 }
 
 // EstimateGas estimates gas usage for the given smart contract call.
-func (e *PublicEthAPI) EstimateGas(args CallArgs, blockNr rpc.BlockNumber) (hexutil.Uint64, error) {
-	result, err := e.doCall(args, blockNr, big.NewInt(emint.DefaultRPCGasLimit))
+func (e *PublicEthAPI) EstimateGas(args CallArgs) (hexutil.Uint64, error) {
```

Comment:
`eth_estimateGas` takes the same parameters as `eth_Call` which includes a QUANTITY TAG (eg. blockNumber).

See [eth_call Parameters](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_call)

Reply:
Resolved as outdated documentation